meeteval/io/base.py

```diff
@@ -160,6 +160,11 @@ def parse(cls, s: str, parse_float=decimal.Decimal) -> 'Self':
         # return cls([cls.line_cls.parse(line) for line in s.splitlines() if line.strip()])
         raise NotImplementedError
 
+    @classmethod
+    def merge(cls, *o) -> 'Self':
+        assert all([type(o_) == type(o[0]) for o_ in o]), o
```

Comment:
Change to `assert all([type(o_) == cls for o_ in o]), o`

Reply:
Done